.gitmodules

```diff
@@ -8,8 +8,8 @@
 	url = https://github.com/llvm/llvm-project.git
 	shallow = true
 	ignore = dirty
-[submodule "enzyme"]
+[submodule "mlir/Enzyme"]
 	path = mlir/Enzyme
-	url = https://github.com/EnzymeAD/Enzyme.git
```

Comment:
Very minor, but since our other URLs are all like that, I was wondering if there is a specific reason for this change?

Reply:
I think I just specified a different protocol when cloning the submodule, updating the repo. It is inconsequential. I can change it back if you prefer.

Reply:
I was just curious since the change must been made intentionally no? It's fine as is.